consumer/hack/bind.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -euo pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+# shellcheck source=consumer/hack/run_test_workload.sh
+source "$SCRIPT_DIR/run_test_workload.sh"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Setup Pipeline Service on a kcp workspace.
+
+Mandatory arguments:
+    -f, --from SERVICE_WORKSPACE
+        Path to the workspace exposing the APIExport.
+
+Optional arguments:
+    -k, --kubeconfig KUBECONFIG
+        Path to the kubeconfig with the connection information to the kcp workspace.
+    -t, --to SERVICE_WORKSPACE
+        Workspace in which to bind the pipeline service.
```

Comment:
Suggestion: User workspace in which Pipeline Service would be bound

Reply:
Reworded, let me know if if is not okay.